test/common.py

```diff
@@ -981,14 +997,14 @@ def get_nodejs(self):
       return None
     return config.NODE_JS_TEST
 
-  def require_node(self):
+  def require_node(self, allow_wasm_engines=False):
```

Comment:
It seems odd to mark a test as `require_node` but also allow wasm engines.   Maybe we can just remove `require_node` from the tests in question instead?

Reply:
Yeah, I wonder about this, too. The original reason I did it like this comes from this line in `also_with_standalone_wasm`: https://github.com/emscripten-core/emscripten/blob/1ae691e9c61efe2d19fa2ab5e7cdd92f17596bf2/test/common.py#L639

Eventually this ends up in `require_engine()` where `self.wasm_engines` is set to an empty list. So what happened is that none of the WASM engines were actually used for any of the `also_with_standalone_wasm` tests!

Alternatively I could stash the `self.wasm_engines` in a local variable, then call `self.require_node()`, and then repopulate `self.wasm_engines`?

Reply:
> Alternatively I could stash the self.wasm_engines in a local variable, then call self.require_node(), and then repopulate self.wasm_engines?

I've implemented this, avoiding this `allow_wasm_engines` parameter.